docs/_docs/components/divider.md

```diff
@@ -49,4 +49,4 @@ Default line style is solid.
 | Name        | Type                                                                   | Default   | Description                                        |
 |-------------|------------------------------------------------------------------------|-----------|----------------------------------------------------|
 | DividerType | [DividerType]({{ "/docs/helpers/enums/#dividertype" | relative_url }}) | `Solid`   | Specifies horizontal line style variants.          |
-| Text        | string                                                                 | null      | Label that will appear between the solid lines.    |
\ No newline at end of file
+| Text        | string                                                                 | null      | Label that will appear between the solid lines.    |
```

Comment:
This change is Github being Github - probably some line ending being changed. Take it or leave it :)

Reply:
I will take it :)